pagefaultgames/pokerogue
# [Fix] Remove and Deactivate Unused Color Palettes to Pass All Tests

## What are the changes?

- Removed unused color palettes from deactivated sprites.
- Deactivated the variant color palette of therian since the back variant color palette was already deactivated.
- Ensured all tests are passing.

### Screenshots/Videos
![webstorm64_PVnaDuKi9B](https://github.com/pagefaultgames/pokerogue/assets/44787002/7b7c44d0-f47a-41e2-b5d1-1c22b6acb68f)


## How to test the changes?




1. Checkout the branch with these changes.
2. Run `npm run test` to ensure all tests are passing.

## Checklist
